src/bitcoin-cli.cpp

```diff
@@ -642,7 +642,7 @@ class NetinfoRequestHandler : public BaseRequestHandler
         "  send     Time since last message sent to the peer, in seconds\n"
         "  recv     Time since last message received from the peer, in seconds\n"
         "  txn      Time since last novel transaction received from the peer and accepted into our mempool, in minutes\n"
-        "           \"*\" - the peer requested we not relay transactions to it (relaytxes is false)\n"
+        "           \"*\" - whether we relay transactions to this peer (relaytxes is false)\n"
```

Comment:
This doesn't seem right. Even if relaytxes is true, this might be false (for example this is an outbound block-relay-only connection?)

Reply:
unrelated: It seems the cli will default to true if the field is not available, which might also be confusing if an outbound block-relay-only connection is spun up.

Reply:
Good catch. Perhaps: `we don't relay transactions to this peer (relaytxes is false or not available)`

Reply:
Proposed a fixup in #26328.

Reply:
Oh, I think I read this as `version.relaytxes` is false, but this means `getpeerinfo.relaytxes` is false? The two are different, so for one interpretation this comment is right and for the other it is wrong?

Reply:
Will clarify next time I touch this that it's the `getpeerinfo "relaytxes" field` (what is `version.relaytxes`?)

Reply:
> what is version.relaytxes?

https://github.com/bitcoin/bips/blob/master/bip-0037.mediawiki#extensions-to-existing-messages

Reply:
> [bitcoin/bips@`master`/bip-0037.mediawiki#extensions-to-existing-messages](https://github.com/bitcoin/bips/blob/master/bip-0037.mediawiki?rgh-link-date=2023-01-11T17%3A12%3A10Z#extensions-to-existing-messages)

Thanks!

Reply:
Thanks! The various namings for tx relay can be a little confusing.